src/utils/colors.ts

```diff
@@ -0,0 +1,66 @@
+import { MonoIconProps } from '@atb/assets/mono-icon';
+
+export function colorToOverrideColor(
```

Comment:
Don't think we need both this file and `utils/color.ts`.

Reply:
My bad, didn't notice after rebase. Fixed now 👍